sn_cli/src/subcommands/wallet/wo_wallet.rs

```diff
@@ -0,0 +1,443 @@
+    Result,
+};
+use sn_client::{Client, ClientEvent};
+use sn_transfers::{
+    CashNoteRedemption, DerivationIndex, MainPubkey, NanoTokens, OfflineTransfer, SignedSpend,
+    UniquePubkey, WalletError, WatchOnlyWallet,
+};
+use std::{
+    collections::{BTreeMap, BTreeSet},
+    path::{Path, PathBuf},
+    str::FromStr,
+};
+use walkdir::WalkDir;
+
+const DEFAULT_RECEIVE_ONLINE_WALLET_DIR: &str = "receive_online";
+const ROYALTY_TRANSFER_NOTIF_TOPIC: &str = "ROYALTY_TRANSFER_NOTIFICATION";
+
+// Please do not remove the blank lines in these doc comments.
+// They are used for inserting line breaks when the help menu is rendered in the UI.
+#[derive(Parser, Debug)]
+pub enum WatchOnlyWalletCmds {
+    /// Print the watch-only wallets addresses.
+    Addresses,
+    /// Print the wallet balance.
+    Balance {
+        /// The hex-encoded public key of an existing watch-only wallet.
+        #[clap(name = "public key")]
+        pk: Option<String>,
+    },
+    /// Deposit CashNotes from the received directory to the chosen watch-only wallet.
+    /// Or Read a hex encoded CashNote from stdin.
+    ///
+    /// The default received directory is platform specific:
+    ///  - Linux: $HOME/.local/share/safe/client/\<pk\>/cash_notes
+    ///  - macOS: $HOME/Library/Application Support/safe/client/\<pk\>/cash_notes
+    ///  - Windows: C:\Users\{username}\AppData\Roaming\safe\client\\<pk\>\cash_notes
+    ///
+    /// If you find the default path unwieldy, you can also set the RECEIVED_CASHNOTES_PATH environment
+    /// variable to a path you would prefer to work with.
+    #[clap(verbatim_doc_comment)]
+    Deposit {
+        /// Read a hex encoded CashNote from stdin.
+        #[clap(long, default_value = "false")]
+        stdin: bool,
+        /// The hex encoded CashNote.
+        #[clap(long)]
+        cash_note: Option<String>,
+        /// The hex-encoded public key of an existing watch-only wallet to deposit into it.
+        #[clap(name = "public key")]
+        pk: String,
+    },
+    /// Create a watch-only wallet from the given (hex-encoded) key.
+    Create {
+        /// Hex-encoded main public key.
+        #[clap(name = "public key")]
+        pk: String,
+    },
+    /// Builds an unsigned transaction to be signed offline. It requires an existing watch-only wallet.
+    Transaction {
+        /// Hex-encoded public key of the source watch-only wallet.
+        #[clap(name = "from")]
+        from: String,
+        /// The number of SafeNetworkTokens to transfer.
+        #[clap(name = "amount")]
+        amount: String,
+        /// Hex-encoded public address of the recipient.
+        #[clap(name = "to")]
+        to: String,
+    },
+    /// Broadcast a transaction that was signed offline.
+    ///
+    /// This command will create the cash note for the recipient.
+    /// This cash note can then be shared with the recipient, who can then
+    /// use the 'deposit' command to use/claim the funds.
+    Broadcast {
```

Comment:
`Broadcast` indicates `broadcast something to live network`
but the above comment indicates the cmd just `create a cash note from signed transaction`

Reply:
yes, but it also mentions in L84 `Broadcast a transaction that was signed offline.`

Reply:
right, then maybe better merge this two sentence into one, say:
This command will create the cash note for the recipient and broadcast it to the network